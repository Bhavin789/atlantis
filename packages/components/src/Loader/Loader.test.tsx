import React from "react";
import renderer from "react-test-renderer";
import { cleanup, fireEvent, render } from "@testing-library/react";
import { Loader } from ".";

afterEach(cleanup);

it("renders a Loader", () => {
  const tree = renderer
    .create(
      <Loader loading determinate={false}>
        Loading
      </Loader>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
