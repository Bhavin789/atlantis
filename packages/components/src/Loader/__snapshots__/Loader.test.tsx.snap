// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a Loader 1`] = `
<div
  className="loader overlay"
>
  Loading
  <div>
    <div
      aria-busy={true}
      aria-label="loading"
      className="spinner"
      role="alert"
    />
  </div>
</div>
`;
